quarto/03-QC-LBT01.qmd

```diff
@@ -0,0 +1,172 @@
+tlg_lbt01 <- chevron::lbt01_main(syn_data, lbl_overall = "Overall ARM")
+head(tlg_lbt01, n = 15) #truncated display
+
+```
+
+### 2. Flatten the table into a data.frame
+
+A {rtables} based output can be flattened into a data frame using the `as_results_df()` function from the {rtables package}. The `make_ard` argument set to `TRUE`, will format the data similar to the output generated by the {cards}/{cardx} package. 
+Setting the `add_tbl_str_decimals` to `FALSE` will not return a column with the statistic as a formatted string.
+
+```{r, warning=FALSE}
+#| code-fold: false
+#| code-summary: "Show the code"
+rtables_result <- rtables::as_result_df(tlg_lbt01, make_ard = TRUE, add_tbl_str_decimals = FALSE)
+```
+
+### 3. Create a comparable ARD
+
+Using the `ard_continuous` function, we will compute the lab results statistics. 
+Notice how there are multiple variables specified by `by`: observations will be grouped in that sequence before calculating the statistics for the variables listed in `variables`. 
+If your table is pruning unobserved levels during calculation, the `by` parameter should be replaced with the `strata` parameter (more detail described in the documentation [here](https://insightsengineering.github.io/cards/latest-tag/reference/ard_continuous.html#arg-by-strata)).
+
+To calculate the statistics for the overall column, the code should look similar to the prior calculation with the "ARM" variable removed from the `by` (or `strata`) parameter (indicating the data is not stratified by "ARM").
+
+We'll keep the stats for the overall column in a separate data.frame to generalize this workflow.
+```{r}
+#| code-fold: false
+#| message: false
+#| code-summary: "Show the code"
+library(cards)
+
+adlb <- chevron::syn_data$adlb
+
+ard_result <- ard_continuous(adlb, 
+                             by = c(ACTARM, PARAMCD, AVISIT), 
+                             variables = c(AVAL, CHG), 
+                             statistic = 
+                               list(everything() ~ continuous_summary_fns(c("N", "mean", "sd", "median", "min", "max"))))
+
+# calculate overall statistics
+ard_overall <- ard_continuous(adlb, 
+                             by = c(PARAMCD, AVISIT), # note arm var is removed
+                             variables = c(AVAL, CHG), 
+                             statistic = 
+                               list(everything() ~ continuous_summary_fns(c("N", "mean", "sd", "median", "min", "max"))))
+```
+
+### 4.Statistics comparison
+
+#### A. {rtables} reformat
+Some reformatting is required to ensure the resulting data.frames are compatible. Most of the reformatting is adjusting the column names to match that of the ARD result.
+
+  1.  rename `group1` and `group1_level`, `ARM` and `ARM` levels accordingly.
+  2.  rename `group2` and `group2_level`, `PARAMCD` and `PARAMCD` levels accordingly.
+  3.  rename `group3` and `group3_level`, `AVISIT` and `AVISIT` levels accordingly.
```

Comment:
It is not clear to me (I know you change the order of the grouping number) but it seems you change the column names